draft-ietf-dnsop-dnssec-kskroll-sentinel.xml

```diff
@@ -184,89 +187,88 @@
       that Bob, Charlie, Dave and Ed will actually look for the absence or
       presence of web resources; instead, the webpage that they load would
       likely contain JavaScript (or similar) which displays the result of the
-      tests. An example of this is at http://www.ksk-test.net. This KSK
-      mechanism does not rely on the web - this method can equally be used by
-      trying to resolve the names (for example, using 'dig') and checking
+      tests, sends the results to Geoff, or both. This sentinel
+      mechanism does not rely on the web: it can equally be used by
+      trying to resolve the names (for example, using the common "dig" command) and checking
       which result in a SERVFAIL.</t>
 
-      <t>[ Note that the KSK Sentinel mechanism measures a very different
-      (and, in our opinion, much more useful!) metric than RFC8145 -- RFC8145
-      relied on resolvers reporting the list of keys that they have -- this
-      doesn't reflect what the *user* impact of the KSK roll will be. As we
-      cannot get perfect visibility into all resolvers, we will have to aim
-      for "do the least harm", not "do no harm" ]</t>
+      <t>Note that the sentinel mechanism described here measures a very different
+      (and likely more useful) metric than <xref target="RFC8145"/>. RFC 8145
+      relies on resolvers reporting the list of keys that they have to root servers. That
+      reflects on how many resolvers will be impacted by a KSK roll, but not
+      what the user impact of the KSK roll will be.</t>
     </section>
 
-    <section title="Sentinel Mechanism">
+    <section title="Sentinel Mechanism in Resolvers">
       <t>DNSSEC-Validating resolvers that implement this mechanism MUST be
       performing validation of responses in accordance with the DNSSEC
       response validation specification <xref target="RFC4035"/>.</t>
 
-      <t>This sentinel mechanism makes use of 2 special labels,
-      "kskroll-sentinel-is-ta-&lt;tag-index&gt;." (intended to be used in a
-      query where the response can answer the question: Is this the key tag a
-      trust anchor which the validating DNS resolver is currently trusting?)
-      and "kskroll-sentinel-not-ta-&lt;tag-index&gt;." (intended to be used in
-      a query where the response can answer the question: Is this the key tag
-      of a key that is NOT in the resolver's current trust store?). The use of
+      <t>This sentinel mechanism makes use of two special labels. The
+      "kskroll-sentinel-is-ta-&lt;tag-index&gt;" label is used in a
+      query where the response can answer whether this the key tag of a
+      trust anchor which the validating DNS resolver is currently trusting.
+      The "kskroll-sentinel-not-ta-&lt;tag-index&gt;" label is used in
+      a query where the response can answer whether this the key tag
+      of a trust anchor that is NOT in currently trusting.</t>
+      
```

Comment:
The last sentence / phrase is hard to parse (" that is NOT in currently trusting."); I'll patch it later, no need up update PR.